components/Feature/TopicExplorer/index.test.js

```diff
@@ -34,16 +34,24 @@ describe('TopicExplorer', () => {
       expect(screen.queryByText('topic two')).toBeNull();
     });
 
+    it('shows autocomletion of search terms', () =>{
+      fireEvent.change(screen.getByLabelText('Try searching for keywords like'), {
+        target: { value: 'f' },
+      });
+      expect(screen.getByText('fourth tag')).toBeInTheDocument
+      expect(screen.getByText('fifth tag')).toBeInTheDocument
```

Comment:
I'd explicitly test here than the _content_ for these tags _isn't_ in the document - think it's cheap and intention-revealing.

```suggestion
      expect(screen.getByText('fifth tag')).toBeInTheDocument
      expect(screen.queryByText('topic one)).toBeNull
```

Reply:
I've updated this as part of the previous commit 😄